src/v/config/configuration.cc

```diff
@@ -480,6 +480,26 @@ configuration::configuration()
        .visibility = visibility::user},
       model::timestamp_type::create_time,
       {model::timestamp_type::create_time, model::timestamp_type::append_time})
+  , log_message_timestamp_alert_before_ms(
+      *this,
+      "log_message_timestamp_alert_before_ms",
+      "Threshold in milliseconds for alerting on messages with a timestamp "
+      "after the broker's time, meaning they are in the future relative to the "
+      "broker's clock.",
+      {.needs_restart = needs_restart::no,
+       .example = "315360000000",
+       .visibility = visibility::tunable},
+      std::chrono::milliseconds::max())
+  , log_message_timestamp_alert_after_ms(
```

Comment:
does this need to be clamped... like at min 3 minutes of delay otherwise a user can shoot themselves on the foot and spam the broker

Reply:
made 5 minutes for both but i can change it